test/Mutation/MutatorTest.php

```diff
@@ -19,8 +19,8 @@ public function testMaxLen() {
                 if ($result === null) {
                     continue;
                 }
-                $this->assertTrue(\strlen($result) <= $maxLen, "$mutator[1]");
+                $this->assertTrue(\strlen($result) <= $maxLen, $mutator[1]);
```

Comment:
This was some lazy debugging on my part ;) It would be better to give this a proper message like `"$mutator[1] violated maximum length constraint"`.

Reply:
Oh.. sorry! I should have noticed that!
Anyways, I added that very same message that you suggested. Thanks!